NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -960,5 +960,54 @@ public enum ServerEventType : int
 		/// Parameters: <see cref="IPlayer"/> player, <see cref="Scp079Camera"/> camera.
 		/// </remarks>
 		Scp079CameraChanged = 115,
+
+		/// <summary>
+		/// Executes when a player looks at SCP-096.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="IPlayer"/> target, <see cref="bool"/> isForLooking.
+		/// </remarks>
+		Scp096AddingTarget = 116,
+
+		/// <summary>
+		/// Executed when a SCP096 gets angered.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="bool"/> clearTime, <see cref="float"/> enragedTimeLeft
+		/// </remarks>
+		Scp096Enraging = 117,
+
+		/// <summary>
+		/// Executed when SCP-096 calms down.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player.
+		/// </remarks>
+		Scp096CalmingDown = 118,
+
+		/// <summary>
+		/// Executed when SCP-096 charges.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player.
+		/// </remarks>
+		Scp096Charging = 119,
```

Comment:
Maybe just merge it into one event (`Scp096RageStateUpdate`?) with 2 args (IPlayer and Scp096RageState)?